package.json

```
@@ -1,14 +1,26 @@
 {
-    "name": "jvar",
-    "version": "1.5.2",
-    "author": "Michael Thornes <zytekaron@gmail.com>",
-    "description": "A utility package for node developers",
-    "main": "src/index.js",
-    "types": "./typings/index.d.ts",
-    "keywords": ["util", "utility"],
-    "license": "GPLV3",
-    "repository": "Zytekaron/jvar.js",
-    "dependencies": {
-        "lodash": "^4.17.15"
-    }
+	"name": "jvar",
+	"version": "1.5.3",
+	"author": "Michael Thornes <zytekaron@gmail.com>",
+	"description": "A utility package for node developers",
+	"main": "./index.js",
+	"module": "./index.mjs",
+	"types": "./typings/index.d.ts",
+	"exports": {
+		"require": "./index.js",
+		"import": "./index.mjs"
+	},
+	"scripts": {
+		"prepublishOnly": "gen-esm-wrapper ./index.js ./index.mjs"
+	},
+	"keywords": [
+		"util",
+		"utility"
+	],
+	"license": "GPLV3",
+	"repository": "Zytekaron/jvar.js",
+	"dependencies": {
+		"gen-esm-wrapper": "^1.1.1",
+		"lodash": "^4.17.15"
+	}
```

```suggestion
	"dependencies": {
		"lodash": "^4.17.15"
	},
   "devDependencies": {
      "gen-esm-wrapper": "^1.1.1"
   }
``` 
This wouldn't be used in prod, would it?